assignments/objects.js

```diff
@@ -19,46 +20,125 @@ const example = {
 
 // Write your intern objects here:
 
+const michael = {
+    "id": 0,
+    "name": "Michael",
+    "email": "email@email.com",
+    "gender": "M"
+  }
+  const Mitzi = {
+    "id": 1,
+    "name": "Mitzi",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+  }
+  const Kennan = {
+    "id": 2,
+    "name": "Kennan",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M",
+    "speak": function(){
+      console.log("Hello, my name is " + this.name)
+    }
+  }
+  const Keven = {
+    "id": 3,
+    "name": "Keven",
+    "email": "kmummery2@wikimedia.org",
+    "gender": "M"
+  }
+  const Gannie = {
+    "id": 4,
+    "name": "Gannie",
+    "email": "gmartinson3@illinois.edu",
+    "gender": "M"
+  }
+  const Antonietta = {
+    "id": 5,
+    "name": "Antonietta",
+    "email": "adaine5@samsung.com",
+    "gender": "F",
+    "multiplyNums": function(a,b){
+      console.log(a*b);
+    }
+  }
+
 
 // ==== Challenge 2: Reading Object Data ==== 
+console.log('==== Objects Challenge 2 ====');
+
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(Mitzi["name"]);
```

Comment:
Are you able to demonstrate the other valid method for accessing a value on your object besides with key notation? :)